cmd/podman/images/list.go

```diff
@@ -192,20 +195,15 @@ func sortImages(imageS []*entities.ImageSummary) []imageReporter {
 	return imgs
 }
 
-func tokenRepoTag(tag string) (string, string) {
-	tokens := strings.Split(tag, ":")
-	switch len(tokens) {
-	case 0:
-		return tag, ""
-	case 1:
-		return tokens[0], ""
-	case 2:
-		return tokens[0], tokens[1]
-	case 3:
-		return tokens[0] + ":" + tokens[1], tokens[2]
-	default:
-		return "<N/A>", ""
-	}
+func tokenRepoTag(ref string) (string, string) {
+
+	repo, _ := reference.Parse(ref)
```

Comment:
I feel like we should convert this to (string, string, error) and return the errors here

Reply:
I added this feature & committed